### DISCUSSION TITLE
[Proposal] Review if it's good to update the Redis compatibility version

### DISCUSSION THREAD

**Discussion by git-hulk** [apache/kvrocks#2151] [General]
Many Redis 6/7 commands have been supported in Kvrocks and our minimal promised compatibility version is 4.0.0. So I would like to review if we should update it.

It's also mentioned in https://github.com/apache/kvrocks/pull/2150#discussion_r1519123723

**Comment by jihuayu** [comment]
Great! Here are some commands that we have not completed yet.

- https://github.com/apache/kvrocks/issues/1942
- https://github.com/apache/kvrocks/issues/1585
- https://github.com/apache/kvrocks/issues/1805

**Reply by git-hulk** [reply]
Yes, maybe we can wait until the last two tracking issues are finished.

**Comment by PragmaTwice** [comment]
Yeah, I think it's also important to find incompatibility in existing commands.

**Reply by git-hulk** [reply]
Yes, I think we can fetch and compare the supported commands between Kvrocks and Redis 6.
